colp_build/sql/name.sql

```diff
@@ -0,0 +1,110 @@
+DROP TABLE IF EXISTS dcas_parcel_names;
+WITH compare AS(
+SELECT
+TRIM(
+	REGEXP_REPLACE(
+		REGEXP_REPLACE(
+		    REGEXP_REPLACE(
+		        REGEXP_REPLACE(
+		            REGEXP_REPLACE(
+		                REGEXP_REPLACE(
+		                    REGEXP_REPLACE(
+		                        REGEXP_REPLACE(
+		                            REGEXP_REPLACE(
+		                                REGEXP_REPLACE(
+		                                    REGEXP_REPLACE(
+		                                        REGEXP_REPLACE(
+		                                            REGEXP_REPLACE(
+		                                                REGEXP_REPLACE(
+		                                                    REGEXP_REPLACE(
+		                                                        REGEXP_REPLACE(
+		                                                            REGEXP_REPLACE(
+		                                                                REGEXP_REPLACE(
+		                                                                    REGEXP_REPLACE(
+		                                                                        REGEXP_REPLACE(
+		                                                                            REGEXP_REPLACE(
+		                                                                                REGEXP_REPLACE(
+		                                                                                    REGEXP_REPLACE(
+		                                                                                        REGEXP_REPLACE(
+		                                                                                            REGEXP_REPLACE(TRIM(parcel_name)||' ', 
+		                                                                                            ' & ', 
```

Comment:
I see you're replacing ' & ' and ' AND ' so I'm not sure why instances like this are happening
'SUBSTAION N&R' to 'SUBSTAION NANDR', which shouldn't happen.  Either  ' & ' need to be replaced with ' AND ' OR both ' & ' and '&' need to be replaced with ' AND '